data/src/generate_tsv_summary.py

```diff
@@ -69,18 +114,20 @@ def main() -> None:
             os.remove(f"{path}/{file_path}")
             continue
         iso639_code = file_path[: file_path.index("_")]
-        transcription_level = file_path[
-            file_path.index("phone") : file_path.index(".")
-        ].capitalize()
-        wiki_name = _handle_wiki_name(
-            languages[iso639_code], file_path, modifiers
-        )
+        transcription_level = _handle_transcription_level(file_path)
+        wiki_name = languages[iso639_code]["wiktionary_name"]
+        filtered = True if "filtered" in file_path else False
```

Comment:
Instead of this you can write:

```filtered = "filtered" in file_path```

The True/False wrapping here is unnecessary.